_posts/2017-08-25-aug-25.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "This Week In Erlang Aug 25"
+description: ""
+date: 2017-08-25
+tags: [erlang, news]
+comments: false
+share: true
+---
```

Comment:
I found 1 misspelled word in this sentence: 
- ---
	- *Suggested Replacements:* -o-, -i-